content/install-ubuntu.md

```diff
@@ -83,11 +83,11 @@ Let's break down what each of the options offer in regards to hardware and softw
 
 - **Install third-party software for graphics and Wi-Fi hardware**
 
-This installs drivers (some non-free) for support for Wi-Fi (Wireless) and GPU support mainly NVIDIA as the amdgpu driver for AMD GPU's is in the kernel.
+This installs drivers (some non-free) for Wi-Fi and GPU support mainly, NVIDIA as the amdgpu driver for AMD GPU's is in the kernel.
```

Comment:
Like I said, the comma would need to go _before_ "mainly," not after. Please update to the suggested wording:

> This installs drivers (some non-free) to provide Wi-Fi/Bluetooth and GPU support (mainly for NVIDIA graphics, as AMD and Intel graphics are supported natively by the Linux kernel).